src_dll/pkcs11_fake.c

```diff
@@ -184,6 +184,12 @@ CK_RV C_GetAttributeValue(CK_SESSION_HANDLE hSession, CK_OBJECT_HANDLE hObject,
 					get_single_attribute_value(pAttribute, &value, sizeof(value), &ret);
 				}
 				break;
+			case CKA_EC_PARAMS:
+				{
+					CK_BYTE value[] = {};
```

Comment:
Can you add a valid value here? You can find some valid encodings [here](https://github.com/ANSSI-FR/opkcs11-tool/blob/cd7a809da94ecb7dba319a5f790f47e3c8bd940a/ecc_helper.ml#L125) for example.

And also can you add a corresponding test in `test_get_attribute_value` in `test_driver.ml`?